src/cli/commands/build.tsx

```diff
@@ -202,7 +202,7 @@ export const UpdatePlanStep: React.FC<UpdatePlanStepProps> = ({
         id: toTrackingPlanId(newTrackingPlan),
         version: newTrackingPlan.version,
         path: trackingPlanConfig.path,
-        trackCalls: events.map<JSONSchema7>(e => ({
+        trackCalls: events.map<JSONSchema7>((e) => ({
```

Comment:
Is it recommended to add brackets here? When my prettier auto prettifies, it removes this bracket and in the rest of the files

Reply:
You can point your prettier extensions to `.prettierrc` file of the project to eliminate inconsistencies.